src/main/java/uk/ac/ebi/ena/webin/cli/WebinCliMessage.java

```diff
@@ -31,6 +31,7 @@
     CLI_UPLOAD_ERROR("Failed to upload files to webin.ebi.ac.uk because of a {0}. "),
     CLI_SUBMIT_ERROR("The submission has failed because of a {0}. "),
     CLI_AUTHENTICATION_ERROR("Invalid submission account user name or password. Please try enclosing your password in single quotes."),
+    CLI_RATELIMIT_ERROR("Cannot submit more than 1 genome within 24 hours for one submission account, sample and study."),

```

Comment:
- The ```CLI_RATELIMIT_ERROR``` enum name is generic but the message is only for the genome context. (the message is specific to a context there is no way around this). So I would change the enum to ```CLI_GENOME_RATELIMIT_ERROR```.